renku/core/models/projects.py

```diff
@@ -69,6 +71,8 @@ class Project(object):
 
     client = attr.ib(default=None, kw_only=True)
 
+    commit = attr.ib(default=None, kw_only=True)
```

Comment:
why is this needed?

Reply:
It was in the signature of `from_yaml(cls, path, client=None, commit=None)` and I've added it here to avoid a validation error. Since `Project` has its own `from_yaml` now I'll remove this from the signature and here.